#include <exception>
using namespace std;

class Bibliotekarz;
class Ksiazka;
class Wypozyczenie;
class Zwrot;

class Zwrot
{
public: void oplata();
};
